tests/foreman/ui/test_host.py

```diff
@@ -921,6 +922,30 @@ def test_positive_search_with_org_and_loc_context(self):
                 self.hosts.search(host.name, _raw_query=host.name))
 
     @tier2
+    def test_positive_search_by_org(self):
+        """Search for host by specifying host's organization name
+
+        :id: a3bb5bc5-cb9c-4b56-b383-f3e4d3d4d222
+
+        :expectedresults: Search functionality works as expected and correct
+            result is returned
+
+        :BZ: 1447958
+
+        :CaseLevel: Integration
+        """
+        host = entities.Host().create()
+        with Session(self) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            self.assertIsNotNone(
+                self.hosts.search(
+                    host.name,
+                    _raw_query='organization = {}'.format(
+                        host.organization.read().name)
```

Comment:
The bug description is searching by organization id, we can add and other test
so that we can cover searching by id and by name

Reply:
>in WebUI, goto Hosts -> All hosts and use filter "organization = Organization_Name"

:)
extra test for id would be a great suggestion for cli test, for ui we usually don't automate pair of id/name, especially when bz description doesn't explicitly tell us to cover it.

Reply:
The bug description is using api and using ord id for searching, so creating an  API  test will be enough to fully cover this.